README.md

```diff
@@ -1,71 +1,120 @@
-vcr
-===
-
 
+# vcr
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-[![cran checks](https://cranchecks.info/badges/worst/vcr)](https://cranchecks.info/pkgs/vcr)
-[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
+[![cran
+checks](https://cranchecks.info/badges/worst/vcr)](https://cranchecks.info/pkgs/vcr)
+[![Project Status: Active – The project has reached a stable, usable
+state and is being actively
+developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 [![R-check](https://github.com/ropensci/vcr/workflows/R-check/badge.svg)](https://github.com/ropensci/vcr/actions/)
 [![codecov](https://codecov.io/gh/ropensci/vcr/branch/master/graph/badge.svg)](https://codecov.io/gh/ropensci/vcr)
-[![rstudio mirror downloads](https://cranlogs.r-pkg.org/badges/vcr)](https://github.com/metacran/cranlogs.app)
-[![cran version](https://www.r-pkg.org/badges/version/vcr)](https://cran.r-project.org/package=vcr)
+[![rstudio mirror
+downloads](https://cranlogs.r-pkg.org/badges/vcr)](https://github.com/metacran/cranlogs.app)
+[![cran
+version](https://www.r-pkg.org/badges/version/vcr)](https://cran.r-project.org/package=vcr)
+
+Easier HTTP testing\! Record HTTP requests and responses on disk and
+replay them for the unit tests of your R package, to make them
+independent from any connection, faster, and more complete. An R port of
+the Ruby gem [vcr](https://github.com/vcr/vcr)
+
+## Elevator pitch
+
+  - **Setup vcr for your package with `vcr::use_vcr()`**
+  - Tweak the configuration to protect your secrets
+  - **Sprinkle your tests with `vcr::use_cassette()` to save HTTP
+    interactions to disk in “cassettes” files**
+  - If you want to test for package behavior when the API errors, edit
+    the cassettes, or use [webmockr](htts://docs.ropensci.org/webmockr)
 
+Now your tests can work without any internet connection\!
 
-Record HTTP requests and responses on disk and replay them for the unit tests of your R package to make them independent from any connection, faster, and more complete. An R port of the Ruby gem [vcr](https://github.com/vcr/vcr)
+[Demo of adding vcr testing to an R
```

Comment:
when my PR to the http testing book is merged, we could link the corresponding chapter too, as it explains the demo better than just looking at a PR diff.

Reply:
sounds good